simulator/src/main/java/com/hazelcast/simulator/coordinator/Coordinator.java

```diff
@@ -96,21 +98,34 @@
     private RemoteClient remoteClient;
     private CoordinatorConnector coordinatorConnector;
 
-    public Coordinator(TestSuite testSuite, ComponentRegistry componentRegistry, CoordinatorParameters coordinatorParameters,
-                       WorkerParameters workerParameters, ClusterLayoutParameters clusterLayoutParameters) {
+    public Coordinator(TestSuite testSuite,
+                       ComponentRegistry componentRegistry,
+                       CoordinatorParameters coordinatorParameters,
+                       WorkerParameters workerParameters,
+                       ClusterLayoutParameters clusterLayoutParameters) {
         this(testSuite, componentRegistry, coordinatorParameters, workerParameters, clusterLayoutParameters,
                 new ClusterLayout(componentRegistry, workerParameters, clusterLayoutParameters));
     }
 
-    Coordinator(TestSuite testSuite, ComponentRegistry componentRegistry, CoordinatorParameters coordinatorParameters,
-                WorkerParameters workerParameters, ClusterLayoutParameters clusterLayoutParameters, ClusterLayout clusterLayout) {
+    Coordinator(TestSuite testSuite,
+                ComponentRegistry componentRegistry,
+                CoordinatorParameters coordinatorParameters,
+                WorkerParameters workerParameters,
+                ClusterLayoutParameters clusterLayoutParameters,
+                ClusterLayout clusterLayout) {
+
+        this.outputDirectory = ensureExistingDirectory(new File(System.getProperty("user.dir"), testSuite.getId()));
+        echoLocal("Output directory: " + outputDirectory.getAbsolutePath());
```

Comment:
Should be done in `logConfiguration()`, so the constructor doesn't get polluted.